edge-rtac/src/main/resources/domain/edge-rtac/features/rtac.feature

```diff
@@ -0,0 +1,156 @@
+    And match ['Available','Checked out'] contains call expectedData response.holdings,'status'
+    #delete created Items
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@DeleteItems')
+
+  Scenario: For a non-periodical/non-serial, return holdings and item information including availability for each instance UUID included in request
+#   1st instance
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostModeOfIssuance')
+    * def createInstanceResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostInstance')
+    * def firstInstanceId = createInstanceResponse.instanceEntityRequest.id
+    * def instanceId = firstInstanceId
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostServicePoint')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostLocation')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostHoldings')
+    * def createFirstItemResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostItem')
+    * def expectedFirstItemId = createFirstItemResponse.itemId
+#   2nd instance
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostModeOfIssuance')
+    * def createInstanceResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostInstance')
+    * def secondInstanceId = createInstanceResponse.instanceEntityRequest.id
+    * def instanceId = secondInstanceId
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostServicePoint')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostLocation')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostHoldings')
+    * def itemStatusName = 'Checked out'
+    * def createSecondItemResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostItem')
+    * def expectedSecondItemId = createSecondItemResponse.itemId
+
+    Given url edgeUrl
+    And path 'rtac?instanceIds=' + firstInstanceId + ',' + secondInstanceId
+    And param apikey = apikey
+    When method GET
+    Then status 200
+    And print response
+    And match response.instances.holdings.length == 2
+    And match [firstInstanceId,secondInstanceId] contains call expectedData response.instances.holdings,'instances'
+    And match [expectedFirstItemId,expectedSecondItemId] contains call expectedData response.instances.holdings,'holdings'
+    And match ['Available','Checked out'] contains call expectedData response.instances.holdings,'status'
+    #deleteItem
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@DeleteItems')
+
+  Scenario:For periodical/serial, return holdings and item information including availability for each instance UUID included in request WHEN &fullPeriodicals=true
+#   1st instance
+    * def modeOfIssuanceId = '068b5344-e2a6-40df-9186-1829e13cd344'
```

Comment:
Is this what distinguishes a periodical from a non-periodical?

How does the implementation identify them, does it use hard-coded modes of issuance?

Reply:
> Is this what distinguishes a periodical from a non-periodical?

Yes , as i am aware of . as per PO :- **Periodical data is a newspaper or magazine**, but going through mod-rtac code i found there is mode of issuance for Instance also which make difference between periodical & non-periodical data.

Reply:
> How does the implementation identify them, does it use hard-coded modes of issuance?

we can not create Serial mode of issuance because it  is already there in Diku tenent , thats why i have hardcoded it .

Reply:
I don't understand this answer.

My question was intended to be: How does edge-rtac (or mod-rtac) determine whether something is a periodical / serial?

Reply:
> Yes , as i am aware of . as per PO :- Periodical data is a newspaper or magazine, but going through mod-rtac code i found there is mode of issuance for Instance also which make difference between periodical & non-periodical data.

Thank you, that's what I was curious about.

Does that mean that the code in mod-rtac hard codes this ID as well?

Reply:
No , mod-rtac does not harcode ID but it hardcode value "Serial".

https://github.com/folio-org/mod-rtac/blob/master/src/main/java/org/folio/mappers/FolioToRtacMapper.java#:~:text=private%20boolean%20isPeriodical,%7D

Reply:
Thanks, that is useful to know.

Please add a comment in the tests that explain that it is the name of the mode of issuance that is important.

Reply:
I think if you are doing one for newspaper material type you should also do one for journal.

Reply:
added "journal" materialType